### PR TITLE
[ADD] Web log followers change

### PR DESCRIPTION
**Pull request by cmsalmeida**
This module create a message every time a follower is added or removed

### CODE REVIEW

File: web_log_followers_change/README.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Description
```
Comment (elicoidal) [inline web_log_followers_change/README.rst:1]: Hi @cmsalmeida 
Thanks for your contribution to the OCA. 
Please follow the guidelines from:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
Please create a proper README file following: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: web_log_followers_change/__init__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline web_log_followers_change/__init__.py:1]: Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: web_log_followers_change/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+##############################################################################
+
+{
+    'name': 'Log every followers change',
+    'version': '8.0.0.1.0',
+    'author': "ThinkOpen Solutions Brasil, Odoo Community Association (OCA)",
+    'category': 'Web',
+    'website': 'http://www.tkobr.com',
+    'license': 'AGPL-3',
+    'images': [],
```
Comment (elicoidal) [inline web_log_followers_change/__openerp__.py:31]: please remove empty keys (x2)

File: web_log_followers_change/model/mail_followers.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    return res
+
+
+class LogChangeMessageFollowers(osv.Model):
+    ''' I define this class so this code will be executed only if module
+        is installed in database. Otherwise it will be always executed.
+    '''
+    _name = 'mail.thread'
+
+    def __init__(self, *arg, **kwargs):
```
Comment (hbrunn) [inline web_log_followers_change/model/mail_followers.py:69]: This has quite some potential side effects and doesn't play well with other modules doing the same thing. Look at https://github.com/hbrunn/social/blob/8.0-mail_follower_custom_notification/mail_follower_custom_notification/models/mail_thread.py for a safer approach

**Comment by hbrunn** [comment]
@cmsalmeida thanks for this module, good idea. Please also add tests, and move your PR to https://github.com/OCA/social

**Comment by cmsalmeida** [comment]
:+1:  thanks a lot

File: web_log_followers_change/model/mail_followers.py (hunk 4 above)
Comment (cmsalmeida) [inline web_log_followers_change/model/mail_followers.py:69]: :+1:

Comment (cmsalmeida) [inline web_log_followers_change/model/mail_followers.py:69]: I'm not getting how that works, this seems the only way, if I do like that I got error stating that original method is not defined... can you point me to some documentation or guidelines on how to do it?
Thanks in advance

Comment (hbrunn) [inline web_log_followers_change/model/mail_followers.py:69]: The documentation is odoo's source code I'm afraid. Starting points are https://github.com/OCA/OCB/blob/8.0/openerp/models.py#L556 and https://github.com/OCA/OCB/blob/8.0/openerp/modules/registry.py#L170.

I'm toying with a mixing that helps with this, you _should_ be able to just grab that file https://github.com/hbrunn/server-tools/blob/8.0-base_encrypted_field/base_encrypted_field/models/base_patch_models_mixin.py and follow the instructions in the comment. Very much alpha, so ymmv

**Pull request closed without merging**